src/rust/jit_instructions.rs

```diff
@@ -5684,7 +5693,26 @@ pub fn instr_F30F10_reg_jit(ctx: &mut JitContext, r1: u32, r2: u32) {
     ctx.builder
         .store_aligned_i32(global_pointers::get_reg_xmm_offset(r2));
 }
-
+pub fn instr_F30F12_mem_jit(ctx: &mut JitContext, modrm_byte: ModrmByte, r: u32) {
+    instr_660F6E_mem_jit(ctx, modrm_byte, r)
```

Comment:
This implementation isn't correct. See https://www.felixcloutier.com/x86/movsldup